### PR TITLE
Fix __repr__ if topology loaded from PDB

### LINKED ISSUE #1287: can't print atoms read with PDB
### Expected behaviour

Can print atoms read from a PDB

### Actual behaviour

Attribute Error that altloc doesn't exist

```
In [3]: print(u.atoms[0])
---------------------------------------------------------------------------
AttributeError                            Traceback (most recent call last)
<ipython-input-3-7346983b787e> in <module>()
----> 1 print(u.atoms[0])

/home/max/anaconda2/lib/python2.7/site-packages/MDAnalysis/core/groups.pyc in __repr__(self)
   2440             me += ' and segid {}'.format(self.segid)
   2441         if hasattr(self, 'altLoc'):
-> 2442             me += ' and altloc {}'.format(self.altloc)
   2443         return me + '>'
   2444 

/home/max/anaconda2/lib/python2.7/site-packages/MDAnalysis/core/groups.pyc in __getattr__(self, attr)
   2425         else:
   2426             raise AttributeError("{cls} has no attribute {attr}".format(
-> 2427                 cls=self.__class__.__name__, attr=attr))
   2428 
   2429     def __repr__(self):

AttributeError: Atom has no attribute altloc


```

### Code to reproduce the behaviour

``` python
import MDAnalysis as mda
from MDAnalysis.datafiles import PDB

u = mda.Universe(PDB)
print(u.atoms[0])
```

### Currently version of MDAnalysis:
(run `python -c "import MDAnalysis as mda; print(mda.__version__)"`)
0.16.0-dev9

### PR DESCRIPTION
**Pull request by kain88-de**
Fixes #1287

Changes made in this Pull Request:
 - repr works again for PDB

PR Checklist
------------
 - [x] Tests?
 - ~~[] Docs?~~
 - ~~[ ] CHANGELOG updated?~~
 - [x] Issue raised/referenced?

### CODE REVIEW

File: testsuite/MDAnalysisTests/core/test_groups.py [hunk 1]
```diff
@@ -893,3 +893,12 @@ def check_operator(op, method, level):
         for op, method in operators:
             for level in levels:
                 yield check_operator, op, method, level
+
+
+class TestAtomGroup(object):
+
+    @staticmethod
+    def test_PDB_atom_repr():
+        u = mda.Universe(PDB)
```
Comment (richardjgowers) [inline testsuite/MDAnalysisTests/core/test_groups.py]: Can we use one of the cheaper PDB universes? This is 47k atoms iirc

Comment (kain88-de) [inline testsuite/MDAnalysisTests/core/test_groups.py]: Which one is smaller? Do we have a `small_PDB`?

Comment (kain88-de) [inline testsuite/MDAnalysisTests/core/test_groups.py]: it's PDB_small

Comment (kain88-de) [inline testsuite/MDAnalysisTests/core/test_groups.py]: Should it be possible to add altloc to makeUniverse?

Comment (kain88-de) [inline testsuite/MDAnalysisTests/core/test_groups.py]: I'm using the make_Universe. I heard you prefer if people use it ;)

File: testsuite/MDAnalysisTests/core/test_groups.py [hunk 2]
```diff
@@ -893,3 +893,11 @@ def check_operator(op, method, level):
         for op, method in operators:
             for level in levels:
                 yield check_operator, op, method, level
+
+
+class TestAtomGroup(object):
```
Comment (richardjgowers) [inline testsuite/MDAnalysisTests/core/test_groups.py]: Can this get put in `test_atomgroup`?

Comment (kain88-de) [inline testsuite/MDAnalysisTests/core/test_groups.py]: Sure? I usually like that filename -> test_filename convention to find tests for functions.

**Pull request merged**